tutorials/Oracles/Oracles.ipynb

```diff
@@ -935,7 +935,7 @@
     "\n",
     "* If $|x\\rangle=|10101\\rangle$ and $|jasmine\\rangle=|01010\\rangle$, do nothing (there is no day on which you both are free).\n",
     "* If $|x\\rangle=|10001\\rangle$ and $|jasmine\\rangle=|01010\\rangle$, flip the state $|y\\rangle$ (you are both free on Wednesday).\n",
-    "* If $|x\\rangle=|00000\\rangle$ and $|jasmine\\rangle=|00000\\rangle$, flip the state $|z\\rangle$ (you are both free all week).\n",
+    "* If $|x\\rangle=|00000\\rangle$ and $|jasmine\\rangle=|00000\\rangle$, flip the state $|y\\rangle$ (you are both free all week).\n",
```

Comment:
Does the corresponding Q# code has y or z?

Reply:
I don't think the user ever sees the backend code, but I did the matching change on the backend